R/Utils.R

```diff
@@ -1,3 +1,5 @@
+#' @import Seurat ggplot2 rlang
+#' @importFrom lsa cosine
```

Comment:
being picky: i dont think you actually need to @importFrom this method. When you use it, you qualify it as lsa::cosine. I actually think that's probably the best way to do this. This avoids needing to load cosine() into our namespace and it cleaner on the whole.

Reply:
I actually dont think you need to @import those either. I have gotten lazy and imported Seurat and ggplot (since they are so common) and therefore I guess this is probably consistent with the rest of the module and fine. but do we need rlang?

Reply:
I'm fuzzy on how this shakes out when building packages. I have two questions here.

1. Is putting lsa in DESCRIPTION sufficient to be able to execute lsa::cosine()? 

2. Is rlang inherently loaded into an R environment? We need sym to pass strings into ggplot/dplyr since aes_string() is depreciated, but I'm not savvy to the history/integration of rlang as a package.

Reply:
Yes, putting lsa in DESCRIPTION is required no matter what.

I think rlang is inherently loaded. It probably doesnt hurt to put it in DESCRIPTION either, but I'm not sure whether that's actually necessary or not.